sdks/python/test-suites/gradle.properties

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Following properties define which Python versions be used when the tests running.
+# Versions must be indicated major and minor versions connected with period.
+# And if you want to use multiple versions, concat them with comma and no whitespace.
+
+# dataflow test-suites
+dataflow_precommit_it_task_py_versions=2.7,3.7
+dataflow_mongodbio_it_task_py_versions=3.5
+dataflow_chicago_taxi_example_task_py_versions=2.7
+
+# direct test-suites
+hdfs_integration_test_task_py_versions=2.7,3.8
+direct_mongodbio_it_task_py_versions=2.7,3.5
```

Comment:
Let's modify https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_PostCommit_Python_MongoDBIO_IT.groovy for now.

Also, it seems like there is a duplication in MongoDBIO suites. I think we can remove the separate jenkins Job for MongoDB IO and include it into the direct runner suites in a separate change.

Also we can add version specifiers for MongoDB performance tests in another change.

Reply:
Updated `beam_PostCommit_Python_MongoDBIO_IT.groovy`

> I think we can remove the separate jenkins Job for MongoDB IO and include it into the direct runner suites in a separate change.

You mean `job_PostCommit_Python_MongoDBIO_IT` and `job_PostCommit_Python_MongoDBIO_Load_Test` are to be moved to direct runner suites? @tvalentyn

Reply:
I think MongoDB IO can run with Direct runner postcommit, similar to HDFS IO. job_PostCommit_Python_MongoDBIO_Load_Test can be a separate job, for which we control the version in settings.gradle.